README.md

```diff
@@ -1041,6 +1041,42 @@ endpoints:
 ```
 
 
+#### Configuring Ses alerts
+| Parameter                      | Description                                                                                | Default       |
+|:-------------------------------|:-------------------------------------------------------------------------------------------|:--------------|
+| `alerting.ses`                 | Settings for alerts of type `ses`                                                          | `{}`          |
+| `alerting.ses.accessKeyId`     | AWS Access Key Id                                                                          | Optional `""` |
+| `alerting.ses.secretAccessKey` | AWS Secret Access Key                                                                      | Optional `""` |
```

Comment:
For consistency, it should be `alerting.aws-ses.access-key-id` and `alerting.aws-ses.secret-access-key` respectively